packages/backend-app-api/config.d.ts

```diff
@@ -33,6 +33,37 @@ export interface Config {
        */
       dangerouslyDisableDefaultAuthPolicy?: boolean;
 
+      /** Controls how to store keys for plugin-to-plugin auth */
+      keyStore?:
+        | { type: 'database' }
+        | {
+            type: 'static';
+            static: {
```

Comment:
How do you feel about the extra `static` wrapping here given the union structure here? I assume this is to keep it similar to the auth backend, but it's already declared in a different way and uses `type` instead of `provider`.

Reply:
we debated it, happy to go either way, it's just one of those things where config merging COULD become a problem in a super hypothetical situation where another type of store is added in the future with the same key that isn't an array but an object. And then you have a base config file with one sort of value and the prod config file with a different one 🤷 But it's so unlikely to happen that it's a bit philosophical. Happy to go either way, what do you prefer?

Reply:
Yep config merging is a good concern, we'd need to make it an array if we go with a shared structure. Let's keep as is 👍